frontend/testing/cypress/src/integration/studio/resourceadm.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+/// <reference types="../../support" />
+
+import * as texts from "@altinn-studio/language/src/nb.json";
+
+// First Cypress tests of sub-repo Resourceadm: this is a work in progress
+
+context("Resourceadm", () => {
+  before(() => {
+    cy.studiologin(Cypress.env("autoTestUser"), Cypress.env("autoTestUserPwd"));
+    // the login session will last throughout the tests
+  });
+
+  beforeEach(() => {
+    cy.visit("/resourceadm/ttd/ttd-resources/"); // each test starts the same place
+  });
+
+  it("is possible to visit Resourceadm main page", () => {
+    cy.url().should("include", "/ttd/ttd-resources"); // URL assertion of main page
```

Comment:
I think we could remove this comment as well, since the cypress code is very self explained here. :)
```suggestion
    cy.url().should("include", "/ttd/ttd-resources");
```